packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -198,6 +198,19 @@ const tooltipStyles = css`
   text-align: center;
 `;
 
+export function displayProductName() {
```

Comment:
FYI another strategy for testing the dates is to do this:

```suggestion
export function displayProductName(today = new Date(Date.now()), ) {
```

Reply:
Even better...